README.md

```diff
@@ -41,8 +41,10 @@ build the generator from scratch.
 ```sh
 $ git clone https://github.com/googleapis/gapic-generator-ruby.git
 $ cd gapic-generator-ruby
-$ gem build gapic-generator.gemspec
-$ gem install gapic-generator-0.1.0.gem
+$ git submodule init
+$ git submodule update
```

Comment:
I think you can simplify this to one line `git submodule update --init`, but it's just a nit.

Reply:
I should have suggested the one-liner in the first place, please update this in subdir README(s) as well.

Reply:
Updated.